doc/advanced-topics.md

```diff
@@ -46,6 +46,64 @@ curl http://${KEYSTONE_HOST}/v3/OS-TRUST/trusts \
 Apart from the generation of the trust, the use of secured Context Brokers should be transparent to the user of the IoT
```

Comment:
Should this PR include a CHANGES_NEXT_RELEASE entry? If I understand correctly the discussion on #961, this PR would be fixing that issue, so it could be mentioned in the CHANGES_NEXT_RELEASE entry.

Reply:
Fixed 51b8f10 + f4ebb69 - CNR updated